spec/components/provider_interface/work_history_and_unpaid_experience_component_spec.rb

```diff
@@ -12,12 +12,14 @@
 
   let(:work_experiences) do
```

Comment:
These are the tests that i was on about. I think this will be fixed by the card you mentioned though @cwrw